models/channel/src/builder.ts

```diff
@@ -6,6 +6,7 @@ import type { TCreateChannelBuilder } from './types';
 
 const Model: TCreateChannelBuilder = () =>
   Builder<Channel>({
+    name: 'Channel',
```

Comment:
There are a bunch of models where I added the name in order to help debugging.

I think is better to keep them.